Package.swift

```diff
@@ -370,6 +370,7 @@ let package = Package(
         "presence",
         "embedded",
         "connections/c",
+	"connections/connectionsd",
```

Comment:
Just noticed a minor tabs/spaces consistency issue while skimming through this PR, also on line 383.

Thanks for that huuuuge effort, I'm really looking forward to the Linux support :)